src/api/getVaultId.js

```diff
@@ -0,0 +1,20 @@
+const validateAssertions = require('../lib/validators/validateAssertions')
+
+module.exports = async (dvf, token, nonce, signature) => {
+  validateAssertions(dvf, {token})
+
+  const existingVaultId = dvf.config.tokenRegistry[token].starkVaultId
+  if (existingVaultId != null) {
+    return existingVaultId
+  }
+  else {
+    const endpoint = '/v1/trading/r/getVaultId'
```

Comment:
not this is not used (probably legacy from before you refactored)

Reply:
yes, thanks!

Reply:
removed